onfido-java/src/main/models/extraction.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Extraction",
+  "path": "extractions",
+  "properties": {
+    "document_id": {
+      "type": "string",
+      "description": "The unique identifier of the uploaded document to run extraction on"
+    },
+    "document_classification": {
+      "$ref": "document_classification.yaml",
+      "description": "The document classification",
+      "readOnly": true
+    },
+    "extracted_data": {
+      "$ref": "extracted_data.yaml",
+      "readOnly": true,
+      "description": "As extraction is done on a best-effort basis, and also information varies across different documents, none of these properties are guaranteed be present in the response."
+    }
+  }
+}
```

Comment:
Always good to configure your IDE to automatically add newlines at the ends of files in my opinion (not a blocker of course)

Reply:
Update, should have a new line on all files now